package.json

```diff
@@ -3,11 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "mongoose": "^5.7.6",
     "react": "^16.10.2",
     "react-alert": "^6.0.0",
     "react-dom": "^16.10.2",
+    "react-loading": "^2.0.3",
+    "react-redux": "^7.1.1",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.2.0"
+    "react-scripts": "3.2.0",
+    "redux": "^4.0.4",
+    "redux-saga": "^1.1.1"
   },
   "scripts": {
     "start": "react-scripts start",
```

Comment:
can add a additional script to start the server, 
what i meant is something like following:
```
 "start-server": "nodemon server.js"
```

and add 
```
"devDependencies": {
    "nodemon": "^1.19.4"
  }
```
in package.json, to make sure your changes are reflected during development, without the need to keep restarting your backend node server again and again